src/factory/__tests__/mongoDBService.test.ts

```diff
@@ -0,0 +1,108 @@
+import * as E from "fp-ts/Either";
+import * as O from "fp-ts/Option";
+import * as TE from "fp-ts/TaskEither";
+import { left, right } from "fp-ts/lib/Either";
+import { Collection, Db, MongoClient } from "mongodb";
+import { ContinuationTokenItem } from "../../capturer/cosmos/utils";
+import * as MongoUtils from "../../capturer/mongo/utils";
+import { mongoDBService } from "../mongoDBService";
+
+const mockMongoClient: MongoClient = {} as MongoClient;
+const mockDatabase: Db = {} as Db;
+const mockCollection: Collection = {} as Collection;
+const mockItem: ContinuationTokenItem = { id: "test" } as ContinuationTokenItem;
+
+jest.mock("../../capturer/mongo/utils");
```

Comment:
Can we remove this mock ?

Reply:
this command is necessary to spyOn methods. If not present, the spy will not work